pubs/commands/note_cmd.py

```diff
@@ -18,7 +19,12 @@ def command(conf, args):
     ui = get_ui()
     rp = repo.Repository(conf)
     citekey = resolve_citekey(rp, args.citekey, ui=ui, exit_on_fail=True)
+    append = args.append
```

Comment:
With append being a common method for lists, I guess we could find a better name for this variable.